db/migrate/033_create_job_definition_revisions.rb

```diff
@@ -0,0 +1,21 @@
+class CreateJobDefinitionRevisions < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :job_definitions, :version
+    remove_column :job_instances, :job_definition_version
+    remove_column :tokens, :job_definition_version
+    remove_column :executions, :job_definition_version
+
+    create_table :job_definition_revisions do |t|
+      t.belongs_to :job_definition, foreign_key: true, null: false
+      t.text :script, null: false
+      t.belongs_to :user, foreign_key: true, null: true
+      t.datetime :changed_at, null: true
```

Comment:
When does `changed_at` become null?

Reply:
Job definitions created before this changes have revision with null `changed_at`.

https://github.com/cookpad/kuroko2/blob/74841d601abf9a0cb0b7191db9f3799f15dd4954/db/migrate/033_create_job_definition_revisions.rb#L18

Reply:
Using `updated_at` as first `changed_at` seems reasonable to me, how do you think?

Reply:
> Using `updated_at` as first `changed_at`

Sorry, I don't understand. What do you mean?

Reply:
I mean

> Kuroko2::JobDefinitionRevision.create(job_definition: definition, script: definition.script) 

to

> Kuroko2::JobDefinitionRevision.create(job_definition: definition, script: definition.script, changed_at: definition.updated_at)

Reply:
I changed in 3d7e1193d34ae9285079bbd0b6c2bc38f353c0fe.